esphome/components/ufire_ec/ufire_ec.cpp

```diff
@@ -103,7 +103,19 @@ void UFireECComponent::write_data_(uint8_t reg, float data) {
   delay(10);
 }
 
-void UFireECComponent::dump_config() {}
+void UFireECComponent::dump_config() {
+  ESP_LOGCONFIG(TAG, "uFire-EC");
+  LOG_I2C_DEVICE(this)
+  LOG_UPDATE_INTERVAL(this)
+  if (this->ec_sensor_ != nullptr)
+    LOG_SENSOR("  ", "EC Sensor", this->ec_sensor_)
+  if (this->temperature_sensor_ != nullptr)
+    LOG_SENSOR("  ", "Temperature Sensor", this->temperature_sensor_)
+  if (this->temperature_sensor_external_ != nullptr)
+    LOG_SENSOR("  ", "Temperature Sensor external", this->temperature_sensor_external_)
```

Comment:
The `if` are not needed as the `LOG_SENSOR` macro includes a check already
```suggestion
  LOG_SENSOR("  ", "EC Sensor", this->ec_sensor_)
  LOG_SENSOR("  ", "Temperature Sensor", this->temperature_sensor_)
  LOG_SENSOR("  ", "Temperature Sensor external", this->temperature_sensor_external_)
```